### PR TITLE
Clarify component/template review process

### PR DESCRIPTION
**Pull request by Scotchester**
@schaferjh @caheberer 

I wanted to update the wording of this new section slightly to make it clear and obvious that the process can be for adding/changing components, too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,14 +219,15 @@ Add the following checklist to an issue when it's time to implement changes:
...
+ - Be sure to include your business or user experience justification for the proposed change.
+ - The issue should include both a quick illustration or demo of how the new use case would be handled by established components/templates (if it can be handled at all) and how the use case would be handled by your update or addition. 
+2. DM lead reviewers and CF.gov Product Owner have 1 week for discussion and consideration of the change. (Comments from others encouraged and welcome!) If consensus is not reached by the end of the week, the CF.gov Product Owner will make a decision.
 3. If approved, add the “Approved concept” and “Documentation needed” tags to the issue and implement the change within your project work. 
-- If moving forward with the change requires modifications to our CMS code, the project team will take the lead with support from the CF.gov Platform team.
+ - If moving forward with the change requires modifications to our CMS code, the project team will take the lead with support from the CF.gov Platform Team.
```
Comment (schaferjh) [inline README.md:230]: do we usually capitalize team? seems awkward to me, but if it's our usual approach, that's fine.

**Comment by schaferjh** [comment]
I made one verrrrrry minor comment, otherwise 👍

File: README.md (hunk 1 above)
Comment (Scotchester) [inline README.md:230]: IMO, in a construction like that, you would. I don't think we have a consistent approach, just my intuition. I think part of it is that you wouldn't replace the phrase "support from the CF.gov Platform Team" with "support from CF.gov Platform", would you? It seems like, if you'd never use the name without the word "team", then "Team" is part of the proper noun.

cf. D&D. You hear both "the Design & Development team" and just "Design & Development".

Comment (Scotchester) [inline README.md:230]: Perhaps I'm overanalyzing it, but hey ;)

/scranfilled

**Pull request merged**